xarray/core/ops.py

```diff
@@ -198,6 +198,24 @@ def func(self, *args, **kwargs):
     return func
 
 
+def rolling_count(rolling):
```

Comment:
This looks generally OK to me, but possibly a cleaner approach would be to make a function that works at the level numpy arrays instead, like bottleneck functions. I think this could be as simple as:
```
def move_count(array, window, min_count=0, axis=-1):
    not_nulls = duck_array_ops.notnull(array)
    return bottleneck.move_sum(not_nulls, window, min_count, axis=axis)
```

The only possibly awkward part about this is that the `min_count` parameter should be handled differently than the standard bottleneck reduce functions. So it it can't be easily consolidated into the existing wrapper for bottleneck functions, don't worry about it.